### PR TITLE
Improve variable and method names

### PR DESCRIPTION
**Pull request by andreban**
- Rename instances of `page` variables to `route`.
- Rename `afterUpdate` method on Shell to `updateState`.

### CODE REVIEW

File: public/js/shell.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ export default class Shell {
     this._states = new Map();
   }
 
-  addState(page, state) {
-    this._states.set(page, state);
+  addState(route, state) {
```
Comment (sebastianbenz) [inline public/js/shell.js]: suggestions: 

* rename `addState` to `setStateForRoute`
* rename `state` to `shellState`

File: public/js/shell.js [hunk 2]
```diff
@@ -49,8 +49,8 @@ export default class Shell {
     tab.classList.remove('activetab');
   }
 
-  afterAttach(page) {
-    const options = this._states.get(page);
+  updateState(route) {
```
Comment (sebastianbenz) [inline public/js/shell.js]: suggestion: s/updateState/onRouteChange/ -> updateState is an implementation detail, onRouteChange makes it more obvious when this should be called.

File: public/js/shell.js (hunk 1 above)
Comment (andreban) [inline public/js/shell.js]: done.

File: public/js/shell.js (hunk 2 above)
Comment (andreban) [inline public/js/shell.js]: done.

**Pull request merged**